### ISSUE TITLE
train: Scanning 'E:\yolo\yolov5-main\train_data\labels\train' images and labels...:   0%|      | 0/938 [00:00<?, ?it/s]  can not go far!

### ISSUE THREAD

**Issue by endeavorhh** [ultralytics/yolov5#6573]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

long time wait,still can not go far,how to solve it?
 (yolov5) E:\yolo\yolov5-main\yolov5>python train.py --img 640 --batch 16 --epochs 200 --data custom.yaml --weights yolov5s.pt --cache
train: weights=yolov5s.pt, cfg=, data=custom.yaml, hyp=data\hyps\hyp.scratch.yaml, epochs=200, batch_size=16, imgsz=640, rect=False, resume=False, nosave=False, noval=False, noautoanchor=False, evolve=None, bucket=, cache=ram, image_weights=False, device=, multi_scale=False, single_cls=False, optimizer=SGD, sync_bn=False, workers=8, project=runs\train, name=exp, exist_ok=False, quad=False, linear_lr=False, label_smoothing=0.0, patience=100, freeze=[0], save_period=-1, local_rank=-1, entity=None, upload_dataset=False, bbox_interval=-1, artifact_alias=latest
github:  YOLOv5 is out of date by 2 commits. Use `git pull` or `git clone https://github.com/ultralytics/yolov5` to update.
YOLOv5  v6.0-238-gb57abb1 torch 1.10.2+cu113 CUDA:0 (NVIDIA GeForce RTX 2060 SUPER, 8192MiB)

hyperparameters: lr0=0.01, lrf=0.1, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=0.05, cls=0.5, cls_pw=1.0, obj=1.0, obj_pw=1.0, iou_t=0.2, anchor_t=4.0, fl_gamma=0.0, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=1.0, mixup=0.0, copy_paste=0.0
Weights & Biases: run 'pip install wandb' to automatically track and visualize YOLOv5  runs (RECOMMENDED)
TensorBoard: Start with 'tensorboard --logdir runs\train', view at http://localhost:6006/
Overriding model.yaml nc=80 with nc=3

                 from  n    params  module                                  arguments
  0                -1  1      3520  models.common.Conv                      [3, 32, 6, 2, 2]
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]
  2                -1  1     18816  models.common.C3                        [64, 64, 1]
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]
  4                -1  2    115712  models.common.C3                        [128, 128, 2]
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]
  6                -1  3    625152  models.common.C3                        [256, 256, 3]
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]
  8                -1  1   1182720  models.common.C3                        [512, 512, 1]
  9                -1  1    656896  models.common.SPPF                      [512, 512, 5]
 10                -1  1    131584  models.common.Conv                      [512, 256, 1, 1]
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 12           [-1, 6]  1         0  models.common.Concat                    [1]
 13                -1  1    361984  models.common.C3                        [512, 256, 1, False]
 14                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 16           [-1, 4]  1         0  models.common.Concat                    [1]
 17                -1  1     90880  models.common.C3                        [256, 128, 1, False]
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]
 19          [-1, 14]  1         0  models.common.Concat                    [1]
 20                -1  1    296448  models.common.C3                        [256, 256, 1, False]
 21                -1  1    590336  models.common.Conv                      [256, 256, 3, 2]
 22          [-1, 10]  1         0  models.common.Concat                    [1]
 23                -1  1   1182720  models.common.C3                        [512, 512, 1, False]
 24      [17, 20, 23]  1     21576  models.yolo.Detect                      [3, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 256, 512]]
Model Summary: 270 layers, 7027720 parameters, 7027720 gradients, 15.9 GFLOPs

Transferred 343/349 items from yolov5s.pt
Scaled weight_decay = 0.0005
optimizer: SGD with parameter groups 57 weight (no decay), 60 weight, 60 bias
train: Scanning 'E:\yolo\yolov5-main\train_data\labels\train' images and labels...:   0%|      | 0/938 [00:00<?, ?it/s]

**Comment by endeavorhh** [comment]
train: /yolo/yolov5-main/train_data/images/train/
val: /yolo/yolov5-main/train_data/images/val/
nc: 3 
names: ['reflective_clothes','other_clothes','hat']

**Comment by glenn-jocher** [comment]
@endeavorhh 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by endeavorhh** [comment]
![yolo_train](https://user-images.githubusercontent.com/98516344/152988786-a8a6c818-29ca-441c-b620-01d46acc9766.jpg)

**Comment by glenn-jocher** [comment]
@endeavorhh training works correctly. Since this this is not reproducible, i.e. there's no action for us to take. See above for providing a minimum reproducible example.

<img width="1496" alt="Screenshot 2022-02-08 at 13 51 52" src="https://user-images.githubusercontent.com/26833433/152990628-25745ced-8d84-4789-aec7-28207a3ba0b2.png">

**Comment by endeavorhh** [comment]
![2022-02-08_232125](https://user-images.githubusercontent.com/98516344/153018907-9488965a-d0cf-496d-a056-aa95339a0067.jpg)
![2022-02-08_232247](https://user-images.githubusercontent.com/98516344/153018928-099c6b91-3d1f-4d6f-9180-846af5b9f62a.jpg)
![2022-02-08_232343](https://user-images.githubusercontent.com/98516344/153018939-2e63f5e8-450e-4ada-b60a-aacae72b75e1.jpg)
![2022-02-08_232432](https://user-images.githubusercontent.com/98516344/153018955-bf3e66d1-cd52-4f62-a1b4-250babaea898.jpg)

**Comment by endeavorhh** [comment]
this sample is simplest train custom data.i don't why stop at scanning images and labels.

**Comment by endeavorhh** [comment]
![2022-02-09_002927](https://user-images.githubusercontent.com/98516344/153031533-3504ce02-dc00-4aec-a619-71954ac846c6.jpg)

**Comment by endeavorhh** [comment]
i debug it , find after step to except, programe stop at scanning.

**Comment by endeavorhh** [comment]
![2022-02-09_004216](https://user-images.githubusercontent.com/98516344/153033930-537dc74e-f5d1-4581-b625-de88c4a12611.jpg)
no response at this step

**Comment by lizhonghan** [comment]
我和你一模一样的显卡，一摸一样的问题，老哥你解决了么

**Comment by glenn-jocher** [comment]
@lizhonghan sorry, I can only respond in English. If you need assistance with YOLOv5, feel free to describe your issue in English and I'll be happy to help!
